app/src/main/java/com/celdevLabs/foodie_app/NewDishFragment.kt

```diff
@@ -137,12 +136,12 @@ class NewDishFragment : Fragment() {
     //TO DO: consolidate save/update dish
     private fun updateDish() {
         val newName: String = binding.etDishName.text.toString()
-        Log.d("NewDishFragment", "new name: ${newName}")
         val newDate: Long = selectedDate
         val location: String = binding.etLocation.text.toString()
         val notes: String = binding.etNotes.text.toString()
+        val rating: Double = binding.dishRatingBar.rating.toDouble()
```

Comment:
Be consistent here. If you're going to use a `double` use a double everywhere. Above you are using a `float`.